src/sentry/options/defaults.py

```diff
@@ -2390,3 +2390,23 @@
     default=[],
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+
+# Options for setting LLM providers and usecases
+register("llm.provider.options", default={}, flags=FLAG_NOSTORE)
```

Comment:
Do you need to configure these with options-automator or change them at runtime? If you have no scenarios where you need to change this at runtime module level constants/dictionaries could be a simpler solution.

Reply:
just thinking through what the easiest way for self hosted users to configure these -- is it right that they can simply modify their config.yml with options and they'll be picked up?

Reply:
>  is it right that they can simply modify their config.yml with options and they'll be picked up?

Yes, option values can be defined in the yml/py config for self-hosted. Self-hosted also has a django settings file in python that they can update. Changes to either the yml or python will require a restart.

Reply:
going with options so config.yml can be source of config for this, confirmed with open source team this makes sense.